deploy/olm-catalog/keycloak-operator/9.0.2/keycloakbackups.keycloak.org.crd.yaml

```diff
@@ -0,0 +1,106 @@
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: KeycloakBackup is the Schema for the keycloakbackups API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: KeycloakBackupSpec defines the desired state of KeycloakBackup
+          properties:
+            aws:
+              description: If provided, an automatic database backup will be created
+                on AWS S3 instead of a local Persistent Volume. If this property is
+                not provided - a local Persistent Volume backup will be chosen.
+              properties:
+                credentialsSecretName:
+                  description: "Provides a secret name used for connecting to AWS
+                    S3 Service. The secret needs to be in the following form: \n     apiVersion:
+                    v1     kind: Secret     metadata:       name: <Secret name>     type:
+                    Opaque     stringData:       AWS_S3_BUCKET_NAME: <S3 Bucket Name>
+                    \      AWS_ACCESS_KEY_ID: <AWS Access Key ID>       AWS_SECRET_ACCESS_KEY:
+                    <AWS Secret Key> \n For more information, please refer to the
+                    Operator documentation."
+                  type: string
+                encryptionKeySecretName:
+                  description: "If provided, the database backup will be encrypted.
+                    Provides a secret name used for encrypting database data. The
+                    secret needs to be in the following form: \n     apiVersion: v1
+                    \    kind: Secret     metadata:       name: <Secret name>     type:
+                    Opaque     stringData:       GPG_PUBLIC_KEY: <GPG Public Key>
+                    \      GPG_TRUST_MODEL: <GPG Trust Model>       GPG_RECIPIENT:
+                    <GPG Recipient> \n For more information, please refer to the Operator
+                    documentation."
+                  type: string
+                schedule:
+                  description: If specified, it will be used as a schedule for creating
+                    a CronJob
+                  type: string
+              type: object
```

Comment:
Not sure whether this is an issue but on this place in deploy/crds/... we have instanceSelect. Should these two files be synced?

Reply:
Thanks for spotting this!